src/locale/locales/ca/messages.po

```diff
@@ -4711,15 +4711,15 @@ msgstr "Ordena les respostes a la mateixa publicació per:"
 
 #: src/components/moderation/LabelsOnMeDialog.tsx:147
 msgid "Source:"
-msgstr ""
+msgstr "Font:"
 
 #: src/lib/moderation/useReportOptions.ts:65
 msgid "Spam"
-msgstr ""
+msgstr "Brossa"
 
 #: src/lib/moderation/useReportOptions.ts:53
 msgid "Spam; excessive mentions or replies"
-msgstr ""
+msgstr "Brosa; excessives mencions o respostes"
```

Comment:
Comparing to the string above (and after checking on Google Translate), I think there's a missing `s` from `Brossa`